uberduck_ml_dev/text/util.py

```diff
@@ -190,6 +191,13 @@ def basic_cleaners(text):
     return text
 
 
+def turkish_cleaners(text):
+    text = text.replace("İ", "i").replace("I", "ı").replace('"', "").replace("'", "")
```

Comment:
whats the purpose of this text replace?

Reply:
Turkish has subtly different rules for letter casing: the lowercase form of `I` is `ı` instead of `i`. The proper™ way to do this would be to set the locale to `tr-TR`, but that either didn't work or was deemed suboptimal by the contributors.

Reply:
...the removal of the quotation marks is unnecessary, though. It was a temporary change by the contributors for training the base model, and I forgot to remove it from the commit.